cmd/profiles.go

```diff
@@ -75,12 +71,10 @@ User profiles can be configured with a "config.yml" file in the profile director
 	profileNewCommand.Flags().String(cobraext.ProfileFromFlagName, "", cobraext.ProfileFromFlagDescription)
 
 	profileDeleteCommand := &cobra.Command{
-		Use:   "delete",
+		Use:   "delete [profile]",
 		Short: "Delete a profile",
+		Args:  cobra.ExactArgs(1),
```

Comment:
:+1: nice, probably we should review all subcommands to check if these options are needed in more places?

Reply:
Yes, it looks like it would be good to add those validations in the other sub-commands.

Here it has been added for stack sub-commands too: https://github.com/elastic/elastic-package/pull/1412